br/README.md

```diff
@@ -1,131 +1,148 @@
 
-# Build BR and backup!
-make build && \
-bin/br backup full --pd pd0:2379 --storage "local:///data/backup/full" \
-    --log-file "/logs/br_backup.log"
+- Save backup data to Amazon S3 or one mounted network disk on all `TiKV-BR` and `TiKV` nodes.
 
-# Let's drop database.
-mysql -uroot -htidb -P4000 -E -e "DROP DATABASE test; SHOW DATABASES;"
+- Allocate sufficient resources for backup and restoration.
 
-# Restore!
-bin/br restore full --pd pd0:2379 --storage "local:///data/backup/full" \
-    --log-file "/logs/br_restore.log"
+TiKV-BR, TiKV nodes, and the backup storage system should provide network bandwidth that is greater than the backup speed. If the target cluster is particularly large, the threshold of backup and restoration speed is limited by the bandwidth of the backup network.
+The backup storage system should also provide sufficient write/read performance (IOPS). Otherwise, the IOPS might become a performance bottleneck during backup or restoration.
+TiKV nodes need to have at least two additional CPU cores and high performance disks for backups. Otherwise, the backup might have an impact on the services running on the cluster.
 
-# How many rows do we get again? Expected to be 100000 rows.
-mysql -uroot -htidb -P4000 -E -e "SELECT COUNT(*) FROM test.usertable"
+### Best practice
+The following are some recommended operations for using `TiKV-BR` for backup and restoration:
+- It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
+- `TiKV-BR` supports restore on clusters of different topologies. However, the online applications will be greatly impacted during the restore operation. It is recommended that you perform restore during the off-peak hours or use rate-limit to limit the rate.
+- It is recommended that you execute multiple backup operations serially. Running different backup operations in parallel reduces backup performance and also affects the online application.
+- It is recommended that you execute multiple restore operations serially. Running different restore operations in parallel increases Region conflicts and also reduces restore performance.
+- `TiKV-BR` supports checksum between `TiKV` cluster and backup files after backup or restore with the config `--checksum=true`. Note that, if checksum is enabled, please make sure no data is changed or `TTL` expired in `TiKV` cluster during backup or restore.
+- TiKV-BR supports [`api-version`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#api-version-%E4%BB%8E-v610-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) conversion from V1 to V2 with config `--dst-api-version V2`. Then restore the backup files to APIV2 `TiKV` cluster
```

Comment:
Suggest to also talk about that the conversion is mainly used for upgrade TiKV storage from `v1` to `v2`.

Reply:
Added, PTAL~